res/css/structures/_UserMenu.scss

```diff
@@ -272,6 +272,9 @@ limitations under the License.
     .mx_UserMenu_iconHome::before {
         mask-image: url('$(res)/img/element-icons/roomlist/home.svg');
     }
+    .mx_UserMenu_iconHosting::before {
+        mask-image: url('$(res)/img/element-icons/roomlist/home.svg');
```

Comment:
Feels a bit strange to re-use the Home icon here, I can't find this in the Figma, do you know where this came from?

Reply:
Have asked design to provide an icon.

Reply:
New icon provided, see screenshot in the pr comments.